### ISSUE TITLE
Instances need a metrics API

### ISSUE THREAD

**Issue by FestivalBobcats** [supergiant/supergiant#59]
For the initial pass, I think just returning both **stats** and **metrics** from Heapster directly will work.

Here are what **stats** on a **pod** (we will drill down to container level in a later pass) look like:

An I-don't-know-statistics-note on [_percentile_](https://en.wikipedia.org/wiki/Percentile): I believe this is the maximum value of the metric, with the highest 5% of values excluded.

```
{
  "uptime": 2269,
  "stats": {
   "cpu-limit": {
    "minute": {
     "average": 1,
     "percentile": 1,
     "max": 1
    },
    "hour": {
     "average": 10,
     "percentile": 10,
     "max": 10
    },
    "day": {
     "average": 10,
     "percentile": 10,
     "max": 10
    }
   },
   "cpu-usage": {
    "minute": {
     "average": 1,
     "percentile": 1,
     "max": 1
    },
    "hour": {
     "average": 10,
     "percentile": 10,
     "max": 10
    },
    "day": {
     "average": 10,
     "percentile": 10,
     "max": 10
    }
   },
   "fs-limit-dev-mapper-vg--ephemeral-ephemeral": {
    "minute": {
     "average": 88347123712,
     "percentile": 88347123712,
     "max": 88347123712
    },
    "hour": {
     "average": 88353013760,
     "percentile": 88353013760,
     "max": 88353013760
    },
    "day": {
     "average": 88353013760,
     "percentile": 88353013760,
     "max": 88353013760
    }
   },
   "fs-usage-dev-mapper-vg--ephemeral-ephemeral": {
    "minute": {
     "average": 143360,
     "percentile": 143360,
     "max": 143360
    },
    "hour": {
     "average": 10485760,
     "percentile": 10485760,
     "max": 10485760
    },
    "day": {
     "average": 10485760,
     "percentile": 10485760,
     "max": 10485760
    }
   },
   "memory-limit": {
    "minute": {
     "average": 2147483648,
     "percentile": 2147483648,
     "max": 2147483648
    },
    "hour": {
     "average": 2147483648,
     "percentile": 2147483648,
     "max": 2147483648
    },
    "day": {
     "average": 2147483648,
     "percentile": 2147483648,
     "max": 2147483648
    }
   },
   "memory-usage": {
    "minute": {
     "average": 1250013184,
     "percentile": 1250013184,
     "max": 1250013184
    },
    "hour": {
     "average": 1254096896,
     "percentile": 1254096896,
     "max": 1254096896
    },
    "day": {
     "average": 1254096896,
     "percentile": 1254096896,
     "max": 1254096896
    }
   },
   "memory-working": {
    "minute": {
     "average": 1249992704,
     "percentile": 1249992704,
     "max": 1249992704
    },
    "hour": {
     "average": 1254096896,
     "percentile": 1254096896,
     "max": 1254096896
    },
    "day": {
     "average": 1254096896,
     "percentile": 1254096896,
     "max": 1254096896
    }
   }
  }
 }
```

And here's what _memory-usage_ **metrics** on a **pod** looks like:

```
{
  "metrics": [
   {
    "timestamp": "2016-04-15T20:48:00Z",
    "value": 1250054144
   },
   {
    "timestamp": "2016-04-15T20:47:00Z",
    "value": 1254096896
   },
   {
    "timestamp": "2016-04-15T20:46:00Z",
    "value": 1254096896
   }
  ],
  "latestTimestamp": "2016-04-15T20:48:00Z"
 }
```

For metrics, I think they have to be fetched individually. For MVP, we could go a long way without metrics (time series graphs).

**Comment by FestivalBobcats** [comment]
Also, I'm 99% sure that units are:

**CPU**: millicore
**RAM / disk**: bytes

**Comment by FestivalBobcats** [comment]
Not fully implemented yet for timeseries data, but still closed by #63
